kieserver/modules/kieserver/module.yaml

```diff
@@ -39,6 +39,9 @@ artifacts:
 - name: "kie-server-services-jbpm-cluster-7.52.0.Final-redhat-00004.jar"
   dest: "/opt/kie/dependencies/jbpm-clustering"
   md5: "44f302877d28e2493fc28ead4dfb6dea"
+- name: "jbpm-event-emitters-kafka-7.52.0.Final-redhat-00007.jar"
```

Comment:
keep the fields on the same order that you have created for the clustering jar above.

Reply:
ok